app/build.gradle.kts

```diff
@@ -0,0 +1,226 @@
+import java.util.*
+
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-parcelize")
+    id("androidx.navigation.safeargs.kotlin")
+    id("realm-android")
+    id("io.sentry.android.gradle")
+    id("de.mannodermaus.android-junit5")
+}
+
+android {
+
+    namespace = "com.infomaniak.drive"
+
+    compileSdk = 34
+
+    ndkVersion = "28.0.13004108"
+
+    defaultConfig {
+        testInstrumentationRunnerArguments["runnerBuilder"] = "de.mannodermaus.junit5.AndroidJUnit5Builder"
+        applicationId = "com.infomaniak.drive"
+        minSdk = 24
+        targetSdk = 34
+        versionCode = 5_05_000_01
+        versionName = "5.5.0"
+
+        setProperty("archivesBaseName", "kdrive-$versionName ($versionCode)")
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        buildConfigField("String", "CLIENT_ID", """"9473D73C-C20F-4971-9E10-D957C563FA68"""")
```

Comment:
Maybe all `""""...""""` in this file should be replaced with `"\"...\""`.

Reply:
Agreed, updated.